web/src/pages/GeneralSettings/GeneralSettings.tsx

```diff
@@ -71,66 +73,85 @@ const GeneralSettingsPage: React.FC = () => {
 
   const { displayName, email, errorReportingConsent, analyticsConsent } = getGeneralSettingsData.generalSettings; // prettier-ignore
   return (
-    <SimpleGrid columns={3} spacing={5}>
-      <Box as="article">
-        <Card px={6} py={9}>
-          <ErrorBoundary>
-            <CompanyInformationForm
-              initialValues={{
-                displayName,
-                email,
-                errorReportingConsent,
-                analyticsConsent,
-              }}
-              onSubmit={values => updateGeneralSettings({ variables: { input: values } })}
+    <>
+      <Flex direction="column" minHeight="80%">
+        <SimpleGrid columns={3} spacing={5}>
+          <Box as="article">
+            <Card px={6} py={9}>
+              <ErrorBoundary>
+                <CompanyInformationForm
+                  initialValues={{
+                    displayName,
+                    email,
+                    errorReportingConsent,
+                    analyticsConsent,
+                  }}
+                  onSubmit={values => updateGeneralSettings({ variables: { input: values } })}
+                />
+              </ErrorBoundary>
+            </Card>
+          </Box>
+        </SimpleGrid>
+      </Flex>
+      <Footer>
+        <Flex spacing={170}>
+          <Box>
+            <Img
+              src={PantherIcon}
+              alt="Panther logo"
+              nativeWidth={94}
+              nativeHeight={20}
+              display="block"
             />
-          </ErrorBoundary>
-        </Card>
-      </Box>
-      <Box as="article">
-        <Card p={6}>
-          <Flex direction="column" spacing={6}>
-            <Heading as="h2" size="x-small" mt={2}>
-              About Panther
-            </Heading>
-            <Flex as="section" align="center" justify="space-between">
-              <Box>
-                <Box color="navyblue-100" fontSize="small" mb={1}>
-                  Plan
-                </Box>
-                <Box fontWeight="medium">Community</Box>
+          </Box>
+          <Flex spacing={9} align="center">
+            <Box as="section">
+              <Box id="plan" as="dt" color="navyblue-100" fontSize="small" mb={1}>
+                Plan
+              </Box>
+              <Box aria-labelledby="plan" as="dl" fontSize="medium">
+                Community
               </Box>
+            </Box>
+            <Box as="section">
               <LinkButton
                 external
+                size="medium"
                 to="https://runpanther.io/pricing/"
                 variantColor="navyblue"
                 variant="outline"
               >
                 Change
```

Comment:
@dimitriszoitas mentioned we should change it
```suggestion
                Get Enterprise
```

Reply:
Can we also entertain the idea of putting it under the label? If it doesn't fit like that, perhaps with a `size="small"`?